src/release/breaking-changes/shortcut-key-event-migration.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Migrate ShortcutActivator and ShortcutManager to KeyEvent system.
+description: >
+  The raw key event subsystem has been superseded by the key event subsystem,
+  and APIs that use RawKeyEvent and RawKeyboard are converted to KeyEvent and
+  HardwareKeyboard.
+---
+
+## Summary
+
+For some time now (years), Flutter has had two key event systems implemented.
+The new system reached parity with the old platform-specific raw key event
+system, and the raw system will be removed. To prepare for that, the Flutter
+APIs that use the old system are being modified, and for a select few of them we
+have decided to make breaking changes in the API in order to preserve the
+quality of the API.
+
+## Context
+
+In the original key event subsystem handling each platform's quirks in the
+framework and in client apps caused overly complex code, and the old system
+didn't properly represent the true state of key events on the system.
+
+So, the new [`KeyEvent`][]-based system was born, and to minimize breaking
+changes, was implemented in parallel with the old system with the intention of
+eventually deprecating the raw system. That time is quickly arriving, and to
+prepare for it, we have made some minimal breaking changes required to preserve
+the quality of the API.
+
+## Description of change
+
+The change modifies the `ShortcutActivator.accepts` method to take a `KeyEvent`
+and `HardwareKeyboard` instead of the previous `RawKeyEvent` and `RawKeyboard`.
+
+The meaning of `ShortcutActivator.accepts` has changed slightly. Before the
+change, it was assumed that `accepts` was only called if
+`ShortcutActivator.triggers` returned null, or if the key event sent to accepts
+had a logical key that was in the `triggers` list. Now it is always called, and
+may use the `triggers` list as a performance improvement, but is not required
+to. Flutter subclasses such as `SingleActivator` and `CharacterActivator`
+already do this.
+
+The change also modifies the `ShortcutManager.handleKeypress` method to take a
+`KeyEvent` instead of `RawKeyEvent`.
+
+## Migration guide
+
+APIs provided by the Flutter framework are already migrated. Migration is
+needed only if you're using any of the methods listed in the previous section.
```

Comment:
nit: as a reader I would appreciate it if this section contains a bullet list of all the effected APIs os I don't have to parse it out of the prose text above (which mixes old and new API).

Reply:
Good point.  Okay, I added the bulleted list to the above section though, since that's where we're talking about what the changes are.